### PR TITLE
feat: click to add address

### PR DESCRIPTION
**Pull request by momentiris**
Click on map now always pops up a tooltip with address name and coordinates. If the last clicked input was start or stop for booking or vehicle, a button that lets you add this position to the corresponding input field will emerge in the tooltip.

### CODE REVIEW

**Review by izabellalarsson — CHANGES_REQUESTED** [review]
The checkbox for the fragile pacel seems to be missing, in the add booking form. Also the part to add a door code.

edit: forget about it. They where found

**Review by izabellalarsson — APPROVED** [review]

File: packages/engine-ui/src/components/CreateVehicle.js [hunk 1]
```diff
@@ -17,23 +16,16 @@ const CreateVehicle = ({ onSubmit }) => {
     volume: '',
     weight: '',
     timewindow: { start: null, end: null },
-    startPosition: { lat: 61.8172594, lon: 16.0561472 },
-    endDestination: '',
+    startPosition: { lat: 61.8172594, lon: 16.0561472, name: '' },
+    endPosition: { lat: '', lon: '', name: '' },
```
Comment (lundeeell) [inline packages/engine-ui/src/components/CreateVehicle.js:20]: lat lon should not be defaulted string, as it is a number in typings

File: packages/engine-ui/src/components/Tooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+  }
+
+  const parseLastFocusedInputToHumanReadable = (
+    lastFocusedInputCode: 'start' | 'end'
+  ) => {
+    switch (location.pathname) {
+      case '/transports/add-vehicle':
+        return tooltipTexts['vehicle'][lastFocusedInputCode]
+      case '/bookings/add-booking':
+        return tooltipTexts['booking'][lastFocusedInputCode]
```
Comment (lundeeell) [inline packages/engine-ui/src/components/Tooltip.tsx:117]: Can we be sure that pathname is any of these two, as we are missing a default in switch/case

File: packages/engine-ui/src/components/forms/eventHandlers.js [hunk 3]
```diff
@@ -22,11 +22,11 @@ const handleContactInputChange = (
...
 }
 
-const handleDropdownSelect = (propertyName, callback) => ({
-  name,
-  lon,
-  lat,
-  county,
+const handleAddressInput = (propertyName, callback) => ({
+  name = '',
+  lon = '',
```
Comment (lundeeell) [inline packages/engine-ui/src/components/forms/eventHandlers.js:27]: lon/lat defaulted to string, but it is a number?

File: packages/engine-ui/src/components/Tooltip.tsx (hunk 2 above)
Comment (momentiris) [reply packages/engine-ui/src/components/Tooltip.tsx:117]: Not really. we just "know" that this code will only  ever run if you are on one of those two paths. I will add a default case.

File: packages/engine-ui/src/components/CreateVehicle.js (hunk 1 above)
Comment (momentiris) [reply packages/engine-ui/src/components/CreateVehicle.js]: Updated.

File: packages/engine-ui/src/components/Tooltip.tsx (hunk 2 above)
Comment (momentiris) [reply packages/engine-ui/src/components/Tooltip.tsx]: Refactored this, please have a look.

File: packages/engine-ui/src/components/forms/eventHandlers.js (hunk 3 above)
Comment (momentiris) [reply packages/engine-ui/src/components/forms/eventHandlers.js]: Now number or undefined

**Review by lundeeell — APPROVED** [review]

**Pull request merged**